app-admin/rex/rex-1.12.2.ebuild

```diff
@@ -0,0 +1,199 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == 9999 ]]; then
+	GITHUB_USER=RexOps
+	GITHUB_REPO=Rex
+	EGIT_REPO_URI="https://github.com/${GITHUB_USER}/${GITHUB_REPO}.git"
+	EGIT_BRANCH="master"
+	EGIT_CHECKOUT_DIR="${WORKDIR}/${PN}-git"
+	VCS_ECLASS="git-r3"
+else
+	# This is intentional to stop perl-module.eclass doing magic things when it
+	# shouldn't. Like making ${S} contain "Rex" when the git clone has "rex"
+	# Also prevents perl-module.eclass provisioning SRC_URI
+	DIST_AUTHOR=FERKI
+	DIST_NAME=Rex
+	KEYWORDS="~amd64 ~x86"
+fi
+inherit bash-completion-r1 perl-module ${VCS_ECLASS}
+
+DESCRIPTION="(R)?ex, the friendly automation framework"
+
+SLOT="0"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DZIL_DEPENDS="
+	dev-perl/Dist-Zilla
+	dev-perl/Dist-Zilla-Plugin-CheckExtraTests
+	dev-perl/Dist-Zilla-Plugin-ContributorsFile
+	dev-perl/Dist-Zilla-Plugin-Git
+	dev-perl/Dist-Zilla-Plugin-Git-Contributors
+	dev-perl/Dist-Zilla-Plugin-MakeMaker-Awesome
+	dev-perl/Dist-Zilla-Plugin-Meta-Contributors
+	dev-perl/Dist-Zilla-Plugin-MetaProvides-Package
+	dev-perl/Dist-Zilla-Plugin-NextVersion-Semantic
+	dev-perl/Dist-Zilla-Plugin-OSPrereqs
+	dev-perl/Dist-Zilla-Plugin-OurPkgVersion
+	dev-perl/Dist-Zilla-Plugin-Run
+	dev-perl/Software-License
+"
+RDEPEND="
+	virtual/perl-Carp
```

Comment:
Please sort, should be easier to read :)

Reply:
Thanks for the feedback, @hydrapolic!

Normally I would sort these by the Gentoo atom names as well. Apparently in this case one of the maintainers (@kentfredric) decided earlier to sort by the canonical names of the related Perl modules instead.

If one or the other ordering is mandatory in Gentoo, please point me to further resources. Personally I'm fine with both as long as we don't decide to change the sorting criteria too often :)

Reply:
Hi @ferki, you're right. Just checking other perl related stuff and it's the same as here. Sorry for the noise :)

Reply:
No worries, and thanks for caring, @hydrapolic! It's good to have you involved. I even learned that it's common in other Perl packages ;)

Reply:
This is also a bit of an issue with reviewing PR's using Github's interface, because the diff of a version bump doesn't show "what changed", but shows the whole ebuild as being "new".

Where what is useful to review is the diff between the "new" ebuild and the previous one.

If Github presented the "version bump" as a proper diff, the diff you'd have reviewed would have only been:

```diff
--- rex-1.12.1.ebuild	2020-08-08 04:41:11.892146874 +1200
+++ rex-1.12.2.ebuild	2020-09-09 19:00:12.275747373 +1200
@@ -30,6 +30,7 @@
 	dev-perl/Dist-Zilla
 	dev-perl/Dist-Zilla-Plugin-CheckExtraTests
 	dev-perl/Dist-Zilla-Plugin-ContributorsFile
+	dev-perl/Dist-Zilla-Plugin-Git
 	dev-perl/Dist-Zilla-Plugin-Git-Contributors
 	dev-perl/Dist-Zilla-Plugin-MakeMaker-Awesome
 	dev-perl/Dist-Zilla-Plugin-Meta-Contributors
@@ -87,6 +88,7 @@
 	test? (
 		virtual/perl-File-Temp
 		dev-perl/Test-Deep
+		dev-perl/Test-Output
 		>=dev-perl/Test-UseAllModules-0.150.0
 		virtual/perl-autodie
 	)
@@ -121,6 +123,7 @@
 		-e '/^Test::PerlTidy =/d' \
 		-e '/^Test::Pod =/d' \
 		-e '/^\[Test::CPAN::Changes\]/{N;d}' \
+		-e '/^\[OptionalFeature/,/^$/d' \
 		-e '/^\[Test::MinimumVersion\]/{N;d}' \
 		-i dist.ini || die "Can't patch dist.ini"
 }
```

( And this is why I like to manually summarize the inter-version diff when I submit my own PR's for others to merge )